README.md

```diff
@@ -1 +1,11 @@
 # LibreJargon
+An open-source jargon demystifier.
+
+## Features:
+- Builtin PDF viewer
+- Automatic jargon highlighting
+- Reading list
+- Highlight a word to get its definition
+
+## How to Build:
+This project uses web-ext. After installing it, run `npm run build` and then `npm run run`. This will open a browser where you can test the extension.
```

Comment:
Also need to have 'npm install' and maybe even tell people to clone the master branch. It should be step by step instructions that will work on someone's random machine here, so this should be expanded.